modules/src/core/ics04_channel/context.rs

```diff
@@ -138,6 +138,48 @@ pub trait ChannelReader {
     }
 }
 
+/// Defines a subset of the `ChannelReader`'s methods that a light-client implementation can access.
+/// A blanket implementation of this trait is provided for all types that implement `ChannelReader`.
+pub trait ChannelMetaReader {
```

Comment:
I don't understand what the `Meta` means in this context. One option is to name it `ChannelReaderLightClient`, and adopt the same convention everywhere (e.g. `LightClientReader` -> `ClientReaderLightClient`). Or perhaps you can think of a better name?

WDYT?

Reply:
Done! https://github.com/informalsystems/ibc-rs/pull/2332/commits/b9d0add23573a3d0a5d1847502ff0de3aa162c5a